data/palantir-three-file/vsac/2022.03.18/output/code_sets.csv

```diff
@@ -1,5 +1,5 @@
-codeset_id,concept_set_name,concept_set_version_title,project,source_application,source_application_version,created_at,atlas_json,is_most_recent_version,version,comments,intention,limitations,issues,update_message,status,has_review,reviewed_by,created_by,provenance,atlas_json_resource_url,parent_version_id,is_draft,enclave_codeset_id,enclave_codeset_id_updated_at
```

Comment:
`True` -> `TRUE`. This may have been a result of Excel. Either way, either way of spelling truthiness is alright for pandas / most everything else.